lib/connectors_sdk/base/extractor.rb

```diff
@@ -115,7 +115,7 @@ def with_auth_tokens_and_retry(&block)
         end
       end
 
-      def yield_document_changes(modified_since: nil)
+      def yield_document_changes(modified_since: nil, break_after_page: false)
```

Comment:
Why add `break_after_page` here?

With async flow in place, I think @artem-shelkovnikov has reverted all the `break_after_page` changes here: https://github.com/elastic/connectors/pull/101

Reply:
It was a merge oversight. Removed; thanks.